model/import/Template/TemplateProcessor.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2021 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoQtiItem\model\import\Template;
+
+use oat\oatbox\service\ConfigurableService;
+use oat\taoQtiItem\model\import\ItemInterface;
+use oat\taoQtiItem\model\import\TemplateInterface;
+use Renderer;
+
+use function Webmozart\Assert\Tests\StaticAnalysis\string;
+
+class TemplateProcessor extends ConfigurableService
+{
+    /** @var Renderer */
+    private $renderer;
+
+    /**
+     * @return string XML
+     */
+    public function process(ItemInterface $item, TemplateInterface $xmlQtiTemplate): string
```

Comment:
No need the comment.

Maybe we could have a 'ItemsTemplateRenderInterface::process()'

And this could be a concrete class `ItemsQtiTemplateRender`, so it is clear for the user that the focus here is QTI.

This class could be responsible to process multiple items based on import result object. Wdyt?

```suggestion
    public function process(ItemInterface $item, TemplateInterface $xmlQtiTemplate): string
```